src/Symfony/Bridge/ProxyManager/Tests/LazyProxy/PhpDumper/ProxyDumperTest.php

```diff
@@ -80,6 +80,37 @@ public function testGetProxyFactoryCode()
         );
     }
 
+    /**
+     * @dataProvider getPrivatePublicDefinitions
+     */
+    public function testCorrectPublicPrivateInstantiation(Definition $definition, string $access)
+    {
+        $definition->setLazy(true);
+
+        $code = $this->dumper->getProxyFactoryCode($definition, 'foo', '$this->getFoo2Service(false)');
+
+        $this->assertStringMatchesFormat(
```

Comment:
Should be one line to be consistent with the code base.

Reply:
Fixed